app/src/main/res/layout/contributors_activity.xml

```diff
@@ -39,15 +47,30 @@
                         app:layout_collapseMode="parallax"
                         app:layout_collapseParallaxMultiplier="0.7">
 
-                        <ImageView
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:layout_gravity="center_horizontal"
-                            android:layout_marginTop="16dp"
-                            android:scaleType="center"
-                            android:scaleX="0.8"
-                            android:scaleY="0.8"
-                            android:src="@mipmap/ic_splash" />
+                        <FrameLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="239dp">
+
+                            <ImageView
+                                android:id="@+id/glove"
+                                android:layout_width="144dp"
+                                android:layout_height="144dp"
+                                android:layout_gravity="center"
+                                android:layout_marginBottom="5dp"
+                                android:visibility="gone"
+                                app:srcCompat="@drawable/thanos_idle" />
```

Comment:
~~but that is this? does it relate to the app somehow?~~

Reply:
Thanos' gauntlet as a symbol of huge power